metrics/influx_test.go

```diff
@@ -0,0 +1,55 @@
+package metrics
+
+import (
+	"fmt"
+	"net/http"
+	"regexp"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/containous/traefik/types"
+	"github.com/stvp/go-udp-testing"
+)
+
+func TestInflux(t *testing.T) {
+	udp.SetAddr(":8089")
+	// This is needed to make sure that UDP Listener listens for data a bit longer, otherwise it will quit after a millisecond
+	udp.Timeout = 5 * time.Second
+
+	influxRegistry := RegisterInflux(&types.Influx{Address: "localhost:8089", PushInterval: "1s"})
+	defer StopInflux()
+
+	if !influxRegistry.IsEnabled() {
+		t.Errorf("InfluxRegistry should return true for IsEnabled()")
```

Comment:
`t.Errorf` -> `t.Fatalf`

And replace `InfluxRegistry should return true for IsEnabled()` by `Influx registry must be enabled`